setup.py

```diff
@@ -0,0 +1,32 @@
+from setuptools import setup, find_packages
+
+with open("README.md", "r") as f:
+    long_description = f.read()
+
+requirements = [
+    "requests",
+    "pytest"]
+
+setup(name="orderlyweb-api",
```

Comment:
The readme lists orderly-web-py, which is also displayed on the pypi page - can you change the main readme to reflect this new name? (https://pypi.org/project/orderlyweb-api/)

Reply:
Yep, good call. Have published to PyPI with that change.